packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -253,6 +253,15 @@ export const globalOptions = {
       'Specify the directory where the log file will be generated.',
     ),
   }),
+  simulateMaxTime: createOption({
```

Comment:
```
simulateMaxTime: createOption({
    key: 'simulateMaxTime' as const,
    prompt: simulatePrompts.simulateMaxTimePrompt,
    validation: z.number().optional(),
    option: new Option(
      '-m, --simulate-max-time <simulateMaxTime>',
      'Specify the maximum time in miliseconds the simulation will run.',
    ),
  }),
```

the name op the options, should match troughout the whole option>
also the first option should be a single character)

if you say -xx whatever within globaloptions this is wrong, and will be solved.
-m instead of -mt since --simulate-max-time already states what it is